main.py

```diff
@@ -7,6 +7,8 @@
 
 bot = Aria()
 
-extensions = []
+extensions = ["manage"]
+for extension in extensions:
+    bot.load_extension(f"cogs.{extension}")
```

Comment:
extensionとして読み込むのが必ず`cogs`ディレクトリにいるとは限らないのでこの書き方はやめたほうがいいかと思います。
```suggestion
extensions = ["cogs.manage"]
for extension in extensions:
    bot.load_extension(extension)
```

Reply:
あー、確かにそうですね
修正します。